### ISSUE TITLE
gf-bot updates should mark previous comments as outdated

### ISSUE THREAD

**Issue by davelab6** [googlefonts/gftools#199]
Github's UX now offers the feature to hide a comment as outdated, so when the gf-bot is posting a series of long FB report comments on a PR thread, it should be possible to mark the older ones as hidden. 

@m4rc1e is this the right repo for the gf-bot code?

**Comment by m4rc1e** [comment]
Yes, this is the correct repo. I'll look into this once Roboto is out the door.
